src/utils/validations.ts

```diff
@@ -33,5 +33,16 @@ const validateRegister = (err: Error) => {
 
   return 'An unexpected error occurred!';
 };
+type userInputsType = {
+  firstName: string;
+  lastName: string;
+};
+const validateEditProfile = (userInputsData: userInputsType) => {
+  const { firstName, lastName } = userInputsData;
+  if (firstName.length === 0 || lastName.length === 0) {
+    return 'Please fill all the fields!';
+  }
+  return '';
+};
```

Comment:
why is the type declaration here, please move it into interfaces or at the top of the page
please leave an empty row between the functions to improve code readability